README.md

```diff
@@ -55,13 +55,14 @@ processor.call(text)
 :emoji_names         - A list of allowed emoji names. (Array<String>)
 :emoji_url_generator - #call'able object that accepts emoji name as argument and returns emoji image URL. (#call)
                        The original implementation is used when the generator returned a falsey value.
+:hostname            - Used to check whether or not each URL of `href` attributes is external site. (String)
```

Comment:
As I mentioned in https://github.com/increments/qiita-markdown/pull/32#discussion_r67280587, if we treat a URL with subdomain of the `hostname` as an internal link (partial match), I think we should not call this parameter `hostname`. Maybe something like `domain`?

What do you think?